streams/pa.m3u

```diff
@@ -25,3 +25,7 @@ http://209.91.213.10:8088/play/a00h
 https://bcovlive-a.akamaihd.net/474b6caf58f8407fb1d1cc1ef64f50ff/us-west-2/6058004209001/playlist_dvr.m3u8
 #EXTINF:-1 tvg-id="TVN.pa" status="blocked",TVN (720p)
 https://bcovlive-a.akamaihd.net/2f670e324b9b46bba7582e919ed90924/us-east-1/6058004209001/playlist.m3u8
+#EXTINF:-1 tvg-id="SerTV.pa" status="blocked",SerTV
+https://stream-04.nyc.dailymotion.com/sec(JtO5HO8PrOFTKKYNhfGSptRqRWG2TU_2ewNFq2_nPDovLDi5lYhAe75e2axNfirI)/dm/3/x81cvsk/s/live-2.m3u8#cell=lnyc
```

Comment:
cannot be used as they'll change.

Reply:
https://www.dailymotion.com/video/x81cvsk